### PR TITLE
Updated get and count methods in DBStorage and FileStorage:

### PR DESCRIPTION
**Pull request by CKCarr**
as well a…s creating tests for get and count in test_db_storage and test_file_storage:
 also updated test_basemodel.py for update and create time failed test

### CODE REVIEW

File: models/engine/file_storage.py
Comment (Jamie897) [inline models/engine/file_storage.py:1]: In the get method, you should use the objects dictionary directly from the all() method, without passing cls as an argument to all(). The all() method should return all objects, and then you can filter the objects by class inside the get method.

You are missing the class method decorator @classmethod before the count method to indicate that it's a class method.

File: models/engine/db_storage.py
Comment (Jamie897) [inline models/engine/db_storage.py:1]: There is an interruption with the count method. It's not returning the count of objects for a specific class or for all objects in storage.

**Review by CKCarr — COMMENTED** [review]
Thank you for looking through.

**Review by Jamie897 — APPROVED** [review]
Approve

**Review by Jamie897 — APPROVED** [review]
Good

**Pull request merged**